src/JsonpCallbackValidator.php

```diff
@@ -1,5 +1,7 @@
 <?php
 
+namespace JsonpCallbackValidator;
```

Comment:
this is a BC break, because you are changing the class name

Reply:
Hi @stof, thank you for your reply.

I've reverted `JsonpCallbackValidator` namespace.